ts/Series/Venn/VennSeries.ts

```diff
@@ -132,6 +133,8 @@ class VennSeries extends ScatterSeries {
      *         Venn diagram
      * @sample {highcharts} highcharts/demo/euler-diagram/
      *         Euler diagram
+     * @sample {highcharts} highcharts/demo/venn-legend-point/
+     *         Venn diagram with a legend listing points
```

Comment:
Since points is the only option for the legend, it suffices to say "Venn diagram with a legend"